pages/common/sed.md

```diff
@@ -2,21 +2,21 @@
 
 > Run replacements based on regular expressions
 
-- replace all occurrences of a string in a file, and print the result
+- replace the first occurrence of a string in a file, and print the result
 
-`sed 's/{{find}}/{{replace}}/g' {{filename}}`
+`sed 's/{{find}}/{{replace}}/' {{filename}}`
 
-- replace all occurrences of a string in a file, and overwrite the file
+- replace all occurrences of a string in a file, overwriting the file
   contents
 
-`sed -i '' 's/{{find}}/{{replace}}/g' {{filename}}`
+`sed --in-place 's/{{find}}/{{replace}}/g' {{filename}}`
```

Comment:
Confirmed! Thanks for helping us rid of ambiguity here, some pages can
really use it.

I'll merge in what you've got once you let me know.

On Mon, Aug 24, 2015, 18:51 Waldir Pimenta <notifications@github.com> wrote:

> In pages/common/sed.md
> <https://github.com/tldr-pages/tldr/pull/293#discussion_r37773264>:
>
> >    contents
> >
> > -`sed -i '' 's/{{find}}/{{replace}}/g' {{filename}}`
> > +`sed --in-place 's/{{find}}/{{replace}}/g' {{filename}}`
>
> If you agree, let's go with that one.
>
> You mean "replace all occurrences of a string in a file, overwriting the
> file (i.e. in-place)" rather than "Replace all occurrences of a string in
> place"?
>
> If so, yeah, it isn't ideal but I don't think it's too bad. Your version
> is shorter but may be ambiguous. Please confirm and I'll perform the
> change. Btw thanks for the discussion :) (I wasn't entirely happy with my
> forced choice either).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tldr-pages/tldr/pull/293/files#r37773264>.
>